src/features/delegation/components/DelegatorsTable.tsx

```diff
@@ -52,7 +54,7 @@ function DelegatorsTableContent({ delegatee }: { delegatee: Delegatee }) {
         {tableData.map((row) => (
           <tr key={row.label}>
             <td className="py-2 font-mono text-sm text-taupe-600">
-              <ShortAddress address={row.label as `0x${string}`} />
+              <ShortAddress address={row.address!} />
```

Comment:
what happens when the address has a name?

Reply:
If you look at the table data, the object is defined as following:
```
      label: address,
      value: fromWei(delegatorToAmount[address]),
      address: normalizeAddress(address),
```

I purely used the `.address!` to prevent type cast as well as having normalized addresses